src/ol_infrastructure/substructure/vault/policies/github/software_engineer.hcl

```diff
@@ -14,6 +14,12 @@ path "secret-concourse/*" {
   capabilities = ["read", "list"]
 }
 
+# To allow Devs to update creds and
```

Comment:
Should this file be part of this PR?

Reply:
I think that came in as a result of my wrangling with trying to pull in your work from main. I just merged the latest into the branch and will see if it's still there.